dns/zone-configs/k8s.io._0_base.yaml

```diff
@@ -274,6 +274,11 @@ oauth2-proxy:
     value: 34.66.218.218
   - type: AAAA
     value: "2600:1900:4000:627b:8000::"
+atlantis:
+  - type: A
```

Comment:
This is in cluster AAA?

Reply:
Ack- Sorry I meant to come back and wound up on the kubebuilder thread through 😴 (not the one in the channel, direct message with me and Fede)